extensions-core/kafka-indexing-service/src/test/java/org/apache/druid/indexing/kafka/KafkaIndexTaskTest.java

```diff
@@ -1011,6 +1012,7 @@ DataSourceMetadata newDataSchemaMetadata()
   }
 
   @Test(timeout = 60_000L)
+  @Ignore
```

Comment:
why are we ignoring all those tests?

Reply:
I ignored some test cases because now, after this change, they take around 7-8 minutes to finish; as it won't return the runtime exception in the first run, it will throw an exception when all retries are exhausted.

Reply:
Changed the number of retries to 2. Now it takes less than 60 seconds to finish.